### DISCUSSION TITLE
multiple commands in cmd

### DISCUSSION THREAD

**Discussion by you-just-got-cyberpunked** [lvjr/tabularray#132] [Q&A]
Hi, is it possible to _nest_ commands in `cmd` ? For example, what should i put as `A` in `cmd={A}` if I wanted to perform `\textbf{\textit{<cell_text>}}`?

I know, this particular example can be bypassed by just creating a new command, I'm just interested if there is a more elegant/general way than this.

```latex
\documentclass{article}

\usepackage{xcolor}
\usepackage{tabularray}

\newcommand{\textbfit}[1]{\textbf{\textit{#1}}}

\begin{document}
   \begin{tblr}{
      hlines, vlines,
      row{1} = {cmd={\textbfit}}
   }
      a & b \\
      c & d \\
   \end{tblr}
\end{document}
```

**Comment by lvjr** [comment]
For font settings, it is better to use `font` key: `row{1} = {font=\itshape\bfseries}`.

**Reply by you-just-got-cyberpunked** [reply]
Awesome, that will do it.
